addons/l10n_fi/__init__.py

```diff
@@ -0,0 +1,23 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#    
+#    OpenERP, Open Source Management Solution
+#
+#    ODOO Addon module by Sprintit Ltd
+#    Copyright (C) 2014 Sprintit Ltd (<http://sprintit.fi>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
```

Comment:
community is lgpl and we should mention it just like this: 

`# -*- coding: utf-8 -*-
# Part of Odoo. See LICENSE file for full copyright and licensing details.`

Just wanted to make sure this is just an oversight.

Reply:
Just an oversight, fixed with the latest commit.